enhancements/sig-architecture/33-detached-deploy-mode/detached-deploy-mode.md

```diff
@@ -0,0 +1,223 @@
+
+#### Architecture of a combined usage of both side
+
+```
+                    +-----------------------------------------------------------------+                                 
+                    | mnagement-cluster                                               |                                 
+                    |                                                                 |                                 
+                    | cluster-manager                         klusterlet              |                                 
+                    | +-------------------+                   +-------------------+   |                                 
+                    | | +-------------------+                 | +-------------------+ |                                 
+                    | | |                 | |<--------------->| |                 | | |                                 
+                    | +-|-----------------+ |                 +-|-----------------+ | |                                 
+                    |   +-------------------+                   +-------------------+ |                                 
+                    +-------|----------------------------------------------|----------+                                 
+                            |                                              |                                            
+                            |                                              |                                            
+                   watch,create,update,delete...            watch,create,update,delete...                               
+                            |                                              |                                            
+                    +-------v-----------+                     +------------v----------+                                 
+                    |hub                |                     |managed cluster        |                                 
+                    |                   |     logically       |                       |                                 
+                    | +--------+        |<------------------->| +--------+            |                                 
+                    | | CRDs...|        |                     | |CRDs... |            |                                 
+                    | +--------+        |                     | +--------+            |                                 
+                    +-------------------+                     +-----------------------+     
+```
+
+A real use case would be: The hub cluster is installed in **private network** while the managed cluster is in public cloud. right now, the managed cluster cannot connect to the hub cluster. **It also means deployments on managed cluster can not apply any resources to the hub cluster directly**.
```

Comment:
@clyang82  Hello, Chunlin, we have added a real use case here, please have a review and see whether it's clear for you. Thanks~

Reply:
@qiujian16 The use case is added now, please also have a review, thanks~

Reply:
Updated this part, the link has outdated.